### PR TITLE
Minor playback improvements and a few icon and string fixes

### PR DESCRIPTION
**Pull request by dkanada**

### CODE REVIEW

File: src/scripts/browserdeviceprofile.js [hunk 1]
```diff
@@ -174,17 +175,15 @@ define(['browser'], function (browser) {
     }
 
     function supportsMpeg2Video() {
-        return browser.orsay || browser.tizen || browser.edgeUwp || browser.web0s;
+        return browser.tizen || browser.orsay || browser.web0s || browser.edgeUwp;
     }
 
     function supportsVc1() {
-        return browser.orsay || browser.tizen || browser.edgeUwp || browser.web0s;
+        return browser.tizen || browser.orsay || browser.web0s || browser.edgeUwp;
```
Comment (JustAMan) [inline src/scripts/browserdeviceprofile.js]: why reorder?

Comment (dkanada) [reply src/scripts/browserdeviceprofile.js]: just to match the order in other places so it's easier to compare

File: src/components/htmlvideoplayer/plugin.js [hunk 2]
```diff
@@ -1764,7 +1759,7 @@ define(['browser', 'require', 'events', 'apphost', 'loading', 'dom', 'playbackMa
     };
 
     HtmlVideoPlayer.prototype.getAspectRatio = function () {
-        return this._currentAspectRatio;
+        return this._currentAspectRatio || "auto";
```
Comment (dkanada) [inline src/components/htmlvideoplayer/plugin.js]: aspect ratio will be shown correctly now

File: src/components/itemcontextmenu.js [hunk 3]
```diff
@@ -70,7 +70,7 @@ define(["apphost", "globalize", "connectionManager", "itemHelper", "appRouter",
                 commands.push({
                     name: globalize.translate("InstantMix"),
                     id: "instantmix",
-                    icon: "shuffle"
+                    icon: "explore"
```
Comment (dkanada) [inline src/components/itemcontextmenu.js]: this will differentiate the shuffle and instant mix actions

File: src/components/itemcontextmenu.js [hunk 4]
```diff
@@ -178,7 +178,7 @@ define(["apphost", "globalize", "connectionManager", "itemHelper", "appRouter",
                 commands.push({
                     name: globalize.translate("EditImages"),
                     id: "editimages",
-                    icon: "edit"
+                    icon: "image"
```
Comment (dkanada) [inline src/components/itemcontextmenu.js]: separate icons for metadata and images

File: src/components/listview/listview.css [hunk 5]
```diff
@@ -35,6 +35,8 @@
 }
 
 .listItem-indexnumberleft {
+    min-width: 2%;
+    text-align: center;
```
Comment (dkanada) [inline src/components/listview/listview.css]: very minor change to center the track numbers for albums

File: src/scripts/browserdeviceprofile.js [hunk 6]
```diff
@@ -113,14 +125,6 @@ define(['browser'], function (browser) {
...
         } else if (format === 'mp2') {
             typeString = 'audio/mpeg';
-        } else if (format === 'ogg' || format === 'oga') {
-
-            // chrome says probably, but seeing failures
-            if (browser.chrome) {
-                return false;
-            }
-            typeString = 'audio/' + format;
-
```
Comment (dkanada) [inline src/scripts/browserdeviceprofile.js]: fixes ogg playback for chrome

File: src/strings/en-gb.json [hunk 7]
```diff
@@ -83,7 +83,7 @@
     "Art": "Art",
     "AsManyAsPossible": "As many as possible",
     "Ascending": "Ascending",
-    "AspectRatio": "Aspect ratio",
+    "AspectRatio": "Aspect Ratio",
```
Comment (dkanada) [inline src/strings/en-gb.json]: all string changes are to capitalize menu entries during video playback

File: src/scripts/browserdeviceprofile.js [hunk 8]
```diff
@@ -77,8 +72,27 @@ define(['browser'], function (browser) {
...
     }
 
-    function canPlayAudioFormat(format) {
+    function supportsAc3(videoTestElement) {
+        if (browser.edgeUwp || browser.tizen || browser.orsay || browser.web0s) {
+            return true;
+        }
+
+        if (browser.osx || browser.iOS) {
+            return false;
```
Comment (anthonylavado) [inline src/scripts/browserdeviceprofile.js]: I was pretty sure that Safari supports AC-3...

**Comment by Bond-009** [comment]
What about the ac3 support for safari?

**Pull request merged**